pfcpiface/parse-far.go

```diff
@@ -106,5 +113,9 @@ func (f *far) parseFAR(farIE *ie.IE, fseid uint64, upf *upf, op operation) error
 		}
 	}
 
+	if (op == update) && (!flag) {
+		return errors.New("No outer header in FAR update")
+	}
+
```

Comment:
Is this the spec or are you coding to one specific implementation of control plane?

Reply:
this is a particular scenario in case where one of the FAR is sent without any outer header update. so, we don't have anything to update. and thus print a error and move on to the next FAR. this caters to the above comment where we have just printed error for FAR parse and moved on.